CodeChat_Server/codechat.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Installs CodeChat
```

Comment:
Create a nice title for this file, like https://github.com/bjones1/CodeChat_system/blob/master/conf.py#L19.